include/net/wifi_credentials.h

```diff
@@ -94,6 +95,7 @@ struct wifi_credentials_enterprise {
  * @param[in] password_buf_len		length of password_buf
  * @param[out] password_len		length of password
  * @param[out] flags			flags
+ * @param[out] flags			channel
```

Comment:
```suggestion
 * @param[out] channel			channel
```
I think this should be channel? fixes the build error.

Reply:
thanks!